xhr/event-upload-progress.htm

```diff
@@ -4,25 +4,30 @@
 <title>XMLHttpRequest: upload progress event</title>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-    <link rel="help" href="https://xhr.spec.whatwg.org/#handler-xhr-onprogress" data-tested-assertations="../.." />
-    <link rel="help" href="https://xhr.spec.whatwg.org/#dom-xmlhttprequest-upload" data-tested-assertations=".." />
-
+<script src="/common/get-host-info.sub.js"></script>
 <div id="log"></div>
 <script>
-  var test = async_test();
-  test.step(function() {
-    var client = new XMLHttpRequest();
-    client.upload.onprogress = test.step_func(function(e) {
-      assert_true(e instanceof ProgressEvent);
-      // This short payload will most likely be sent before the first progress evt
-      assert_equals(e.loaded, 22);
-      assert_equals(e.total, 22);
-      test.done();
-    });
-    client.onreadystatechange = test.step_func(function() {
-      if (client.readyState === 4) assert_unreached("onprogress not called.");
-    });
-    client.open("POST", "resources/upload.py");
-    client.send("This is a test string.");
-  });
+const remote = get_host_info().HTTP_REMOTE_ORIGIN + "/XMLHttpRequest/resources/corsenabled.py",
```

Comment:
I think /XMLHttpRequest/ should be /xhr/ here.

Reply:
Thanks, I must have rebased this after the rename and forgotten about this or some such.